google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -210,6 +214,11 @@ public String getAppProfileId() {
     return appProfileId;
   }
 
+  /** Returns if channels will periodically reconnect to GFE */
+  public boolean isRefreshingChannel() {
+    return isRefreshingChannel;
+  }
```

Comment:
I would drop the getter...I think it will add needless complexity once you fix the toBuilder() bug

Reply:
nvm, it already correct